Messenger/Messenger.Core/Services/MessageService.cs

```diff
@@ -192,13 +192,23 @@ public static async Task<bool> DeleteMessage(uint messageId)
             LogContext.PushProperty("SourceContext", "MessageService");
             logger.Information($"Function called with parameters messageId={messageId}");
 
+            // Delete replies to prevent errors
             var replies = await RetrieveReplies(messageId);
 
```

Comment:
could you add null checks here? foreach would throw error if reactions == null

Reply:
Fixed in 07a20b9